validators_instructions.md

```diff
@@ -0,0 +1,139 @@
+# Validators Instructions for Setting Up Steward
```

Comment:
This section is similar to the one in the top level readme, so if you update that one let's reflect it here too.

Reply:
done!